wie-geht-studium/lehrbuecher.tex

```diff
@@ -81,7 +80,76 @@ \subsubsection{Mathematische Methoden}
 \end{description}
 
-\subsubsection*{Nachschlagewerke}
+\subsubsection{Informatik allgemein}
+
+In den Informatik-Vorlesungen wird allgemein sehr wenig bewusst Literatur verwendet,
+sondern hauptsächlich die Vorlesungsfolien und vor allem Online-Dokumentationen,
+insbesondere zu den Programmiersprachen, verwendet. Meistens werden gute Wikipedia-Artikel,
+Youtube-Tutorials, Stackoverflow-Posts und Google-Books bzw. Google-Scholar-Links näher angeschaut. 
+
+\subsubsection{Praktische Informatik und Programmierkurs}
+Für den Python-Programmierkurs sollte die Online-Dokumentation angeschaut werden:
+\url{https://docs.python.org/3/tutorial/}
+Für die praktische Informatik und für den Programmierkurs in C++ empfiehlt sich folgende Literatur
+\begin{description}
+\item[Bastian]{
+	ist ein Vorlesungs-Skript, was es auch bei uns im Fachschaftsraum kostenlos in gedruckter Form abgeholt werden kann. In der Praktischen Informatik wird meistens darauf verwiesen oder eine Abwandlung davon verwendet. Hier wird Schritt für Schritt die Programmiersprache C++ an zahlreichen praktischen Beispielen aus dem Themenbereich des wissenschaftlichen Rechnens erklärt, und zeigt somit im Ansatz die Schnittmenge zwischen den Disziplinen Mathematik und Informatik. Das Skript von Herrn Prof. Bastian lässt sich unter folgenden Link herunterladen: \url{https://conan.iwr.uni-heidelberg.de/old-site/teaching/info1_ws2014/info1-skript.pdf}}
+	
+	% Muss noch naeher erklaert werden, ob es fuer praktische und fuer IPK gut ist
+\item[Stroustroup: The C++ Programming Language]{
+	ist das Referenzwerk für die Programmiersprache C++. Ähnlich wie in einer Online-Dokumentation werden hier die Syntax, verschiedene Befehle und Funktionen anhand von Minimalbeispielen näher erklärt.}
+
+\item[Knuth: The Art of Computer Programming]{ist eine große Bandreihe vom berühmten TeX-Erfinder. Das erste Band beschreibt mit mathematischer Strenge die Performanz unterschiedlicher Datenstrukturen. Dabei spielt der Begriff der Laufzeitkomplexität eine große Rolle.}
+
+\end{description}
+
+\subsubsection{Algorithmen und Datenstrukturen}
+\begin{description}
+
+\item[Sedgewick: Algorithms]{
+	ist eine sehr einsteigerfreundliche Lektüre, wo verschiedene Algorithmen eingeführt und anhand einer Java-Implementation näher erläutert werden. Auf die eigene Implementation verschiedener Datenstrukturen wird hier leider nicht eingegangen, sondern es wird auf die in den Bibliotheksfunktionen bereits vorhandenen Strukturen zurückgegriffen. Die Programmbeispiele in Java sind sehr simpel gehalten und lassen sich somit relativ einfach in C++ oder in Python umformulieren. Dieses Buch kann vorweg als Begleitlektüre für die Vorlesung Praktische Informatik genommen werden und aufgrund der Java-Beispiele bereitet es den Leser ebenfalls auf die Vorlesung Einführung in Software Engineering vor.} 
```

Comment:
"ist eine sehr einsteigerfreundliche Lektüre, wo..." ist Umgangssprache. lieber ", in der"?